test/csv_test.py

```diff
@@ -174,3 +174,45 @@ def test_use_existing_csv_file_with_auto_columns():
     assert db2.get_client("01") == client
 
     db1.drop_database()
+
+def test_empty_packets_list():
+    columns = ["id", "packets"]
+    client = {"id": "01", "packets": []}
+
+    db1 = CSVDatabase("dbtest.csv", columns)
+    db1.add_client(client)
+
+    db2 = CSVDatabase("dbtest.csv")
+    assert db2.get_client("01") == client
+
+    db1.drop_database()
+def test_non_empty_packets_list():
+    columns = ["id", "packets"]
+    client = {"id": "01", "packets": ['a','b']}
+
+    db1 = CSVDatabase("dbtest.csv", columns)
+    db1.add_client(client)
+
+    db2 = CSVDatabase("dbtest.csv")
+    assert db2.get_client("01") == client
+
+    db1.drop_database()
+def test_get_all_clients_with_list():
```

Comment:
Rename to `test_get_all_clients_with_packets_list` (other tests you added follow this convention)

Reply:
removed tests